softwerkskammer/test/members/membersService_test.js

```diff
@@ -226,6 +227,8 @@ describe('MembersService', function () {
       getImageFromAvatarProvider = sinon.stub(avatarProvider, 'getImage', function (anyMember, callback) {
         callback(gravatarData);
       });
+      sinon.stub(galleryService, 'storeAvatar').yields(null, 'filename.jpg');
+      sinon.stub(galleryService, 'retrieveScaledImage').yields(null, imagePath);
```

Comment:
Ich meine, Du hast hier einen Test, der vorher gegen den GalleryService integriert hat, so verändert, dass er jetzt einen Stub nimmt. OK, nehme ich mal so hin.

Aber wo ist ein entsprechender Test, der jetzt immer noch gegen den GalleryService integriert? *Das* will ich doch in meinen Tests wissen, und diese Prüfung hast Du hier entfernt. (Oder ich bin blind und habe nicht gesehen, wo Du einen neuen Test eingefügt hast.)